spyder/plugins/editor.py

```diff
@@ -631,6 +632,15 @@ def get_plugin_actions(self):
         self.register_shortcut(self.new_action, context="Editor",
                                name="New file", add_sc_to_tip=True)
 
+        add_shortcut_to_tooltip(self.new_action, context="Editor",
+                                name="New file")
+        
+        self.open_last_closed = create_action(self, _("O&pen last closed"),

```

Comment:
And that's because you named the method associated with this action exactly as the action, so we need to disambiguate them :-)